app/app/src/main/java/com/globant/calculator/android/MainActivity.java

```diff
@@ -15,6 +35,97 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        presenter = new CalculatorPresenter(new CalculatorModel(), new CalculatorView());
+        ButterKnife.bind(this);
+        presenter = new CalculatorPresenter(new CalculatorModel(), new CalculatorView(this));
+    }
+
+    @Override
+    protected void onResume() {
```

Comment:
if this is not necessary, please remove it

Reply:
done